source/drivers/community-supported-drivers.txt

```diff
@@ -158,7 +156,7 @@ Community Supported Drivers Reference
 - Perl
 
   - `Mango <https://metacpan.org/pod/Mango>`_ - A non-blocking
-    pure-Perl driver based on `Mojolicious <http://mojolicio.us>`_
+    pure-Perl driver based on `Mojolicious <https://https://mojolicious.org/>`_
```

Comment:
this link is broken: 

when I click it it actually links to `https://https//mojolicious.org/` for some reason

Reply:
Whoops, copypasta

Reply:
Fixed